proxy/http.go

```diff
@@ -86,8 +86,19 @@ func NewRequestBuilderMiddleware(remote *config.Backend) Middleware {
 		}
 		return func(ctx context.Context, request *Request) (*Response, error) {
 			r := request.Clone()
-			r.GeneratePath(remote.URLPattern)
-			r.Method = remote.Method
+
+			if remote.URLPattern == "/*path" {
+				r.Path = r.Params["Path"]
+			} else {
+				r.GeneratePath(remote.URLPattern)
+			}
+
+			if remote.Method == "ANY" {
+				r.Method = request.Method
+			} else {
+				r.Method = remote.Method
+			}
+
```

Comment:
Putting the correctness of the idea, this lines are not required because the remote method is always equal to the request method...

think about it: the request method is used by the router to find a handler. handlers are registered with the method defined in the config. so, what's the value of the `request.Method` when you are in a handler configured for `GET` requests (just as an example)?